geopolars/src/geoseries.rs

```diff
@@ -624,6 +629,25 @@ impl GeoSeries for Series {
         }
     }
 
+    #[cfg(feature = "proj")]
+    fn to_crs(&self, from: &str, to: &str) -> Result<Series> {
+        use proj::{Proj, Transform};
+
+        let proj = Proj::new_known_crs(from, to, None).unwrap();
+        let output_vec: Vec<Geometry> = iter_geom(self)
+            .map(|mut geom| {
+                // geom.tranform modifies `geom` in place.
+                // Note that this doesn't modify the _original series_ because iter_geom makes a
+                // copy
+                // https://docs.rs/proj/latest/proj/#integration-with-geo-types
+                geom.transform(&proj).unwrap();
```

Comment:
Do we want to handle this error? Also more generally, if  an operation on a member of a Series fails, do we want the entire operation to fail ? I have wondered about that for a few different methods I implemented

Reply:
Yeah we definitely want to handle the error. I'm planning to come back and do a general overhaul to try and remove as many `unwrap`s as possible.

> Also more generally, if an operation on a member of a Series fails, do we want the entire operation to fail 

I lean towards yes, at least to start with. I think that mimics geopandas operations that fail...?

Reply:
Yeah I think it makes sense. There might be cases where returning a None might make sense. If the operation hasn't failed per-say but just isn't defined for a given geometry but yeah for now, it should be an explicit failure for the entire call I think